packages/near-api-js/yarn.lock

```diff
@@ -0,0 +1,6199 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I don't think this should exist under individual packages. Yarn workspaces/Lerna will maintain a single `yarn.lock` at the root of the directory. You'll need to delete this and probably run `lerna bootstrap` or `yarn` from the root to regenerate.